base/tps/shared/conf/db.ldif

```diff
@@ -41,6 +41,12 @@ objectClass: groupOfUniqueNames
 cn: Administrators
 description: Administrators for TPS
 
+dn: cn=Auditors,ou=Groups,{rootSuffix}
+objectClass: top
+objectClass: groupOfUniqueNames
+cn: Auditors
+description: Auditors for TPS
```

Comment:
Actually, in other subsystems the description is "People who can read the signed audits". We probably want to make it consistent.

Reply:
Up to you, doesn't really matter though. The description for other entries aren't consistent either :)

Reply:
ACK, thanks!